Feature/hide scenario creation (#411)
DLR-SC/ESID: docs/changelog/changelog-en.md

- Instead of hard-coded organizations, now organization dropdown options are populated by fetching the actual realms.
- There is more robust validation for the application state. If anything goes wrong during validation, a waring is shown in the bottom-left corner.
- The case data and baseline scenarios are now always shown and the first two scenarios.

> - Non LHA User now cannot access to the Create Scenario button

Reviewer: ```suggestion
- Only logged in users now have access to the "Create Scenario" button.
```

And a German translation:
- Nur eingeloggte Nutzer haben Zugriff auf den "Szenario Erstellen" Button.
